proto/ibc/applications/interchain_accounts/v1/query.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package ibc.applications.interchain_accounts.v1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "ibc/applications/interchain_accounts/v1/account.proto";
+
+option go_package = "github.com/cosmos/ibc-go/modules/apps/27-interchain-accounts/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  // Query to get the address of an interchain account
+  rpc InterchainAccount(QueryInterchainAccountRequest) returns (QueryInterchainAccountResponse) {}
```

Comment:
Maybe rename to `InterchainAccountAddress`?

Reply:
Agree.

Reply:
Updated